lib/install.go

```diff
@@ -74,7 +74,7 @@ func install(tfversion string, binPath string, installPath string, mirrorURL str
 	}
 
 	/* check if selected version already downloaded */
-	installFileVersionPath := ConvertExecutableExt(filepath.Join(installLocation, VersionPrefix+tfversion))
+	installFileVersionPath := ConvertExecutableExt(filepath.Join(installLocation, product.GetVersionPrefix()+tfversion))
```

Comment:
readability nit:
```suggestion
	installFileVersionPath := ConvertExecutableExt(filepath.Join(installLocation, product.GetVersionPrefix() + tfversion))
```

Reply:
VSCode does this for me! If I change, it just changes it straight back! (I assumed it was a common golang styling thing)

Reply:
I also resolved other commit suggestions of the same kind.

Reply:
Thank you :)